guildwars2/guild/general.py

```diff
@@ -213,9 +242,67 @@ class GeneralGuild:
                         inline=False)
                     counter += 1
         if counter == 0:
-            return await ctx.send(
-                "No stash log entries yet for {}".format(guild_name.title()))
+            return await ctx.send("No stash log entries yet for {}".format(
+                guild_name.title()))
         try:
             await ctx.send(embed=data)
         except discord.Forbidden:
             await ctx.send("Need permission to embed links")
+
+    @guild.command(name="default")
+    @commands.cooldown(1, 10, BucketType.user)
```

Comment:
Use a `@commands.guild_only()` decorator like [here](https://github.com/Maselkov/GW2Bot/blob/7b080a17550d379cb4070d3cf91796b93af72d79/guildwars2/notifiers.py#L15) to ensure this command is only usable in servers, then you can skip checking if the command was issued in server

Reply:
Changed for !guild default and removed guild(server)check